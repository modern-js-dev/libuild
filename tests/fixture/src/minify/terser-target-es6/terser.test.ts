import { expect, getLibuilderTest } from '@modern-js/libuild-test-toolkit';

describe('fixture:minify', () => {
  it('when `minify` is `terser`', async () => {
    const bundler = await getLibuilderTest({
      root: __dirname,
      target: 'es6',
      minify: 'terser',
      
    });
    await bundler.build();
    const jsOutput = bundler.getJSOutput();
    const jsChunk = Object.values(jsOutput);
    expect(jsChunk.length === 1).to.be.true;
    expect(jsChunk[0].contents).toMatchSnapshot();
  });
});
