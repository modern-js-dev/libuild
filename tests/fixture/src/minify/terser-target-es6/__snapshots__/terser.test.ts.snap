// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixture:minify when \`minify\` is \`terser\` 1`] = `
"// index.ts
var this_is_a_long_function_name3 = () => {
};
console.log(this_is_a_long_function_name3);
"
`;
